offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -105,6 +107,7 @@ class OfferToPDFConverter implements OfferExporter {
             return this.name;
         }
     }
+    private final Map productGroupClasses = [:]
```

Comment:
add a comment here
```suggestion
/* This map represents the grouping of the different product categories in the offer pdf */
    private final Map productGroupClasses = [:]
```

Reply:
also maybe indicate which values the map holds e.g. Map<ProductGroups,List<Product>>

Reply:
Will do 😄